src/utils/helpers/methodsForStorage.ts

```diff
@@ -0,0 +1,24 @@
+import { Category, Priority, TodoType } from "constants/types";
+
+export const addTodosLocalStorage = (
+  todos: TodoType[],
+  todosDone: TodoType[]
+) => {
+  localStorage.setItem("todos", JSON.stringify(todos));
+  localStorage.setItem("todosDone", JSON.stringify(todosDone));
+};
+
+export const deleteTodosLocalStorage = () => localStorage.clear();
+
+export const checkIfDateExists = () => {
```

Comment:
Change name because actual name suggest that you will receive a boolean value instead of object.